server/app/routes/api/v1/assessments/assessments.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
```

Comment:
Your naming style above was annotations/index.js why is it different for the assessments routes?